src/testoptix/testoptix.cpp

```diff
@@ -122,6 +122,8 @@ void getargs(int argc, const char *argv[])
                 "--debug", &debug1, "Lots of debugging info",
                 "--debug2", &debug2, "Even more debugging info",
                 "--runstats", &runstats, "Print run statistics",
+                "--saveptx", &saveptx, "Save the generated PTX",
+                "--warmup", &warmup, "Perform a warmup launch",
```

Comment:
Just for my edification: what's a warmup launch and under what circumstances is it helpful?

Reply:
There is overhead associated with the first CUDA launch that I wanted to isolate from the actual rendering time, to get a better sense of what kind of performance to expect in an interactive context.

Reply:
Total aside: is there someplace I can learn more about this... how big the "first batch" penalty is, and ways to mitigate it? Is it first batch per app? Per kernel?